scripts/dev-e2e.sh

```diff
@@ -1,41 +1,32 @@
 #!/bin/bash
 
-set -euo pipefail
+#set -euo pipefail
```

Comment:
is this supposed to be commented out?

Reply:
No, it shouldn't - good catch - I needed to see some failures verbosely and `-u` causes Bash to instantly, quietly exit. Fixed in commit 4c655cf.